packages/@uppy/companion/src/server/provider/zoom/index.js

```diff
@@ -28,7 +28,7 @@ class Zoom extends Provider {
     return 'zoom'
   }
 
-  list (options, done) {
+  async list (options, done) {
```

Comment:
yes, thsi is now updated to remove async/await.  Main changes here are:
- moving the 2nd api call into its own method (`_meetingInfo` and `_recordingInfo)
- move calls to these into the `.request` callback (and remove promise.all since we have access to both user response and meeting or recording response now)
- reorder the if/else statement in the `.request` callback to match the user flow abit more closely (eg user clicks into zoom provider, no filters set, then clicks into a month (from/to set) then clicks into a specific meeting (meetingId is set)